spec/safe_statements_spec.rb

```diff
@@ -1279,6 +1279,41 @@ def up
           end
         end
 
+        describe "raw_remove_check_constraint" do
+          before(:each) do
+            setup_migration = Class.new(migration_klass) do
+              define_method(:up) do
+                unsafe_create_table :foos
+                unsafe_add_column :foos, :bar, :text
+                unsafe_add_check_constraint :foos, "bar IS NOT NULL", :name => :constraint_foo_bar_is_not_null
+              end
+            end
+            setup_migration.suppress_messages { setup_migration.migrate(:up) }
+          end
+
+          ["constraint_foo_bar_is_not_null", :constraint_foo_bar_is_not_null].each do |constraint_name|
+            it "does not throw an error when #{constraint_name.is_a?(Symbol) ? "symbol" : "string"} is used for contraint name" do
+              migration = Class.new(migration_klass) do
+                define_method(:up) do
+                  raw_remove_check_constraint :foos, name: constraint_name
+                end
+              end
+
+              RSpec::Matchers.define_negated_matcher :not_raise_error, :raise_error
+              expect do
+                migration.suppress_messages { migration.migrate(:up) }
+              end.to not_raise_error
```

Comment:
Do we need to explicitly check if it doesn't raise an error? It would be _slightly_ less explicit in the output, but if we remove this expectation and it does raise an error, the test should still fail, right?

Reply:
yeah, good point! Removed the check here: https://github.com/braintree/pg_ha_migrations/pull/102/commits/bbc9ac5abe41bd0c1c5aafcd90ab1c559fdb22ec